router/pkg/config/config.schema.json

```diff
@@ -1243,6 +1243,16 @@
               "description": "The maximum size of the request headers. Setting this to 0 uses the default value from the http standard lib, which is 1MiB.",
               "default": "0MiB",
               "bytes": {}
+            },
+            "compression": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "enabled": {
```

Comment:
`compression: enabled`
doesn't really seem intuitive to me. It seems more like we're doing the compression when in reality we're checking whether there is compression and decompressing accordingly.

Reply:
`decompression: enabled` would be my suggestion for that reason